HW4.6.java

```diff
@@ -0,0 +1,57 @@
+package com.company;
+
+public class Person {
+
+    private String name;
+    private String surname;
+    private int age;
+    private int height;
+    private int weight;
+
```

Comment:
better add private constructor

Reply:
Ok. I did it, but i don't understand why and how does it work. Can you explain me?